README.md

```diff
@@ -90,3 +90,8 @@ In *live* mode additionally one can use:
 * `set_consumption(current: float)` – sets auto-consumption (use positive values for consumption)
 
 * `set_temp_outside(temp: float)` – sets temperature outside
+
+* `set_ev_driving(driving_power: float, time_until_charged_h: Optional[float])` - sets EV driving power and optionally 
```

Comment:
Please update README according to new changes

Reply:
Done